src/main/java/ladder/domain/Line.java

```diff
@@ -0,0 +1,48 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class Line {
+    private final List<Bar> bars;
+    private final int SUBTRACT_FIRST_BAR = -1;
+
+    public Line() {
+        bars = new ArrayList<>(List.of(new Bar(false)));
+    }
+
+    public void addBars(int peopleSize, BarGenerator barGenerator) {
+        IntStream.range(0, peopleSize + SUBTRACT_FIRST_BAR)
```

Comment:
SUBTRACT를 빼고 FIRST_BAR 의미만 나타내도 되지 않을까요?
```suggestion
        IntStream.range(0, peopleSize - FIRST_BAR)
```

Reply:
그게 가독성면에서 더 좋은 것 같습니다